packages/client/src/wishlists/__fixtures__/deleteWishlistItem.fixtures.ts

```diff
@@ -1,41 +1,16 @@
-import join from 'proper-url-join';
-import moxios from 'moxios';
-import type { Wishlist, WishlistItem } from '../types';
+import { rest, RestHandler } from 'msw';
+import type { Wishlist } from '../types';
 
 export default {
-  success: (params: {
-    wishlistId: Wishlist['id'];
-    wishlistItemId: WishlistItem['id'];
-    response: Wishlist;
-  }): void => {
-    moxios.stubRequest(
-      join(
-        '/api/commerce/v1/wishlists',
-        params.wishlistId,
-        'items',
-        params.wishlistItemId,
-      ),
-      {
-        response: params.response,
-        status: 200,
-      },
-    );
-  },
-  failure: (params: {
-    wishlistId: Wishlist['id'];
-    wishlistItemId: WishlistItem['id'];
-  }): void => {
-    moxios.stubRequest(
-      join(
-        '/api/commerce/v1/wishlists',
-        params.wishlistId,
-        'items',
-        params.wishlistItemId,
-      ),
-      {
-        response: 'stub error',
-        status: 404,
-      },
-    );
-  },
+  success: (response: Wishlist): RestHandler =>
+    rest.delete(
+      '/api/commerce/v1/wishlists/:wishlistId/items/:wishlistItemId',
```

Comment:
Since this is repeated, in both success and failure, what about isolating this `path` to a `const`? (Applicable to all fixtures)

Reply:
Done :)